<launch>

  <arg name="flip" default="false" />     <!-- Horizontal + Vertical flip -->
  <arg name="numberOfCameras" default="2" />
  <arg name="primaryCamMode" default="false"/>
  <!-- upload calibration to fpga -->
  <arg name="setCalibration" default="true" />
  <!-- use of the AIT ROS message warapper -->
  <arg name="AITMsgs" default="false" />
  <!-- calibrationMode 0 no calibration, 1: calibration for 1 camera pair,
      2: calibration for 2 camera pairs and so on -->
  <arg name="calibrationMode" default="0" />
  <arg name="depthMap" default="true" />
  <!-- 1: 2 front cameras, 3: 2 front + 2nd camera pair,
       7: 2 front + 2nd and 3rd camra pair -->
  <arg name="cameraConfig" default="1" />
  <!-- calibration file in folder calib -->
  <arg name="cameraConfigFile" default="/media/camera/calibration.yaml" />

  <!-- camera node -->
  <node pkg="uvc_ros_driver" name="uvc_camera_two" type="uvc_ros_driver_node" output="screen">
    <param name="flip" type="bool"  value="$(arg flip)" />
    <param name="numberOfCameras" type="int"  value="$(arg numberOfCameras)" />
    <param name="primaryCamMode" type="bool" value="$(arg primaryCamMode)"/>
    <param name="AITMsgs" type="bool"  value="$(arg AITMsgs)" />
    <param name="setCalibration" type="bool"  value="$(arg setCalibration)" />
    <param name="calibrationMode" type="bool"  value="$(arg calibrationMode)" />
    <param name="depthMap" type="bool"  value="$(arg depthMap)" />
    <param name="cameraConfig" type="int"  value="$(arg cameraConfig)" />
    <param name="cameraConfigFile" type="string"  value="$(arg cameraConfigFile)" />
    <rosparam command="load" file="$(find uvc_ros_driver)/params/homography_mapping.yaml" />
  </node>

<remap from="uvc_camera" to="uvc_camera_one"/>

</launch>


